src/maps/mixins.py

```diff
@@ -14,19 +21,26 @@ class LayerViewMixin:
     """A mixin to get the Layer object based on layer_slug in url kwargs."""
 
     def setup(self, *args, **kwargs) -> None:
+        """Setup the mixin."""
         super().setup(*args, **kwargs)
         self.layer = get_object_or_404(Layer, slug=self.kwargs["layer_slug"])
 
-    def get_context_data(self, *args, **kwargs):
+    def get_context_data(self, *args, **kwargs) -> dict:
+        """Get context data."""
         context = super().get_context_data(*args, **kwargs)
         context["layer"] = self.layer
         return context
 
 
 class LayerMapperViewMixin(LayerViewMixin):
```

Comment:
what is a LayerMapper, not a term we use elsewhere is it?

Reply:
Except we do use it for the user permission to change team related map things: from settings.py: "mapper": "Team mapper - layers and feature management",